client/src/components/TicketsSidebar/index.jsx

```diff
@@ -1,42 +1,37 @@
-import React, { Component } from 'react';
-import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
 import './style.css';
 
-class TicketsSidebar extends Component {
-  render() {
-    return (
-      <div className="ticket-sidebar">
-        <button className="ticket-sidebar__new-ticket">
-          <i className="fas fa-plus" />
-          {'          '}
-          Ticket
-        </button>
-        <div className="ticket-sidebar__links-container">
-          <Router>
-            <div className="ticket-sidebar__links">
-              <Link to="/tickets/all-ticket" className="ticket-sidebar__all-link">
-                All
-                {' '}
-              </Link>
-              <i className="far fa-circle" />
-            </div>
-            <div className="ticket-sidebar__links">
-              <Link to="/tickets/my-ticket" className="ticket-sidebar__my-link">My</Link>
-              <i className="far fa-circle" />
-            </div>
-            <div className="ticket-sidebar__links">
-              <Link to="/tickets/drafts-ticket" className="ticket-sidebar__drafts-link">Drafts</Link>
-              <i className="far fa-circle" />
-            </div>
-            <div className="ticket-sidebar__links">
-              <Link to="/tickets/trash-ticket" className="ticket-sidebar__trash-link">Trash</Link>
-              <i className="far fa-circle" />
-            </div>
-          </Router>
+const TicketsSidebar = ({ selected }) => (
+  <div className="ticket-sidebar">
+    <button className="ticket-sidebar__new-ticket">
+      <i className="fas fa-plus" />
+      {'          '}
+      Ticket
+    </button>
+    <div className="ticket-sidebar__links-container">
+      <Router>
+        <div className={`ticket-sidebar__links  ${selected === 'all' ? 'selected-tab' : ''}`}>
+          <Link to="/tickets/all-ticket" className={`ticket-sidebar__all-link ${selected === 'all' ? 'ticket-sidebar__all-link-selected' : ''}`}>
+            All
+            {' '}
+          </Link>
+          <i className={`far fa-circle  ${selected === 'all' ? 'fa-circle-selected' : ''}`} />
         </div>
-
-      </div>
-    );
-  }
-}
+        <div className={`ticket-sidebar__links  ${selected === 'my' ? 'selected-tab' : ''}`}>
```

Comment:
@Jamalat-shamallakh If you'd like, you might be able to turn this link rendering into a map (so you don't have to repeat so much code for each link)
At the top of render, create a variable to map over:
```
const links = [
  {name: 'my', label: 'My'},
  {name: 'drafts', label 'Drafts'},
  {name: 'trash', label: 'Trash'},
]
```
And then in the jsx, map:
```
{links.map({ name, label } => (
  <div className={`ticket-sidebar__links ${selected===name ? 'selected-tab' : ''}`}>
    <link to={`tickets/${name}-ticket` className={...} />
    <i className={...
  </div>
))}
```

That will also make it faster to add new links in the future.